deployment/deployment.go

```diff
@@ -61,6 +74,9 @@ func Create(label, ws, releaseName, cloudRole, executor, airflowVersion string,
 		vars["airflowVersion"] = airflowVersion
 	}
 
+	if dagDeploymentType == "volume" && nfsLocation != "" {
+		vars["dagDeployment"] = map[string]string{"nfsLocation": nfsLocation, "type": dagDeploymentType}
```

Comment:
Is there a way to get a test to cover this line?

Reply:
on it.

Reply:
@adam2k done.